engo_js.go

```diff
@@ -223,6 +223,8 @@ func cancelAnimationFrame(id int) {
 	dom.GetWindow().CancelAnimationFrame(id)
 }
 
+// RunPreparation is called only once, and is called automatically when calling Open
```

Comment:
Noted, I copied it from the engo_glfw.go RunPrep doc to be consistent across platforms.